src/Engine/Task/ObservationZones/Boundary.hpp

```diff
@@ -33,6 +33,13 @@ class OZBoundary : public std::forward_list<GeoPoint> {
   /**
    * Generate boundary points for the arc described by the parameters.
    * This excludes the points at the start/end angle.
+   *
+   * The arc is generated in an increasing angle order - i.e. CW.
```

Comment:
Adding more documentation is good, but not hidden inside this unrelated patch.

Reply:
So, just a question on the logistics:  For the purposes of the PR should this commit be on a branch of its own or should it be broken out as a separate on this branch - i.e. 'ticket_1687_on_master'?

Reply:
My stance: can be same branch in same PR. As long as it's a separate commit, I can easily cherry-pick good commits and postpone the others.